va/va.h

```diff
@@ -761,6 +761,13 @@ typedef enum
      * attribute value \c VAConfigAttribValMaxFrameSize represent max frame size support   
      */
     VAConfigAttribMaxFrameSize           = 38,
+    /** \brief inter frame prediction directrion attribute. Read-only.
+     * this attribute conveys the prediction direction (backward or forword) for specific config
+     * the value could be  VA_PREDICTION_DIRECTION_XXXX. it can be combined with VAConfigAttribEncMaxRefFrames
+     * to describe reference list , and the prediction direction. if this attrib is not present, or value == 0,
+     * frame prediction can be deduced by VAConfigAttribEncMaxRefFrames.
```

Comment:
How is the frame prediction deduced? It would be better to detail it in the comment. 

You defined VA_PREDICTION_DIRECTION_PREVIOUS (1) and VA_PREDICTION_DIRECTION_FUTURE (2) only, is the value set to 0?

Reply:
if this attrib is not present, or value == 0,
     * frame prediction can be deduced by VAConfigAttribEncMaxRefFrames.